s3/pkg/service/multipart.go

```diff
@@ -461,6 +468,7 @@ func (s *s3Service) CompleteMultipartUpload(ctx context.Context, in *pb.Complete
 		Type:             ObjectTypeNormal,
 		Tier:             multipart.Metadata.Tier,
 		Size:             totalSize,
+		EncSize:          totalSize,
```

Comment:
Hi @rhsakarpos , could you please explain what EncSize means, if it is the same as Size, is it necessary?

Reply:
typically, depending on the algorithm used, the encryption on a byte[ ] changes the size of the byte[ ]. This size is needed when we decrypt the encrypted byte[ ]

Reply:
OK, Eh.... but why EncSize equals Size?

Reply:
in this streaming encryption algorithm, the size is the same